### DISCUSSION TITLE
Watcher Software Service Introduction | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#2050] [Q&A]
# Watcher Software Service Introduction | Seeed Studio Wiki

This article provides a detailed setup guide for configuring the Watcher device in three different processing deployment flows,including Cloud Efficient Processing Flow, Hybrid Smart Processing Flow, and Local Secure Processing Flow. It offers an overview of the Watcher software service framework, illustrating the interactions between the user, SenseCraft Mate app, and Watcher device.

https://wiki.seeedstudio.com/watcher_software_service_framework/

**Comment by shin-sforzando** [comment]
I think the abbreviation for “Speech-to-Text” should be STT.

**Reply by nfs0619** [reply]
Thanks for the feedback. We have made the necessary correction.
